bin/genome-env

```diff
@@ -89,7 +104,14 @@ function main {
 
     if test -z "$DISABLE_DB"
     then
-        log.info "Test DB Snapshot: $DB_SNAPSHOT_NAME"
+        if test -n "$DB_ID"
+        then
+            log.info "Using existing test DB"
+            log.info "Test DB Host: ${TESTDBSERVER_DB_HOST}"
+            log.info "Test DB Port: ${TESTDBSERVER_DB_PORT}"
```

Comment:
I think it would be fine to always show the host and port, i.e. move them down with "Test DB Name".